src/symmetry.jl

```diff
@@ -425,23 +425,23 @@ end
 """
 Symmetrize the Hubbard occupation matrix according to the l quantum number of the manifold.
 """
-function symmetrize_hubbard_n(hubbard_n::Array{Matrix{Complex{T}}}, 
-                              model, symmetries, positions) where {T}
+function symmetrize_hubbard_n(manifold::OrbitalManifold,
+                              hubbard_n::Array{Matrix{Complex{T}}},
+                              model, symmetries) where {T}
```

Comment:
This clashes a bit with the argument order and interface of the other symmetrisation functions. I'd say first `model`, then `manifold` then `hubbard_n`, `symmetries` a kwarg ?